.github/workflows/pull_request.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     - name: checkout
       uses: actions/checkout@v3
     - name: build and test
-      run: make frontend && git diff --exit-code
+      run: make frontend i18n && git diff --exit-code
```

Comment:
As `build-frontend` already run `fmt-frontend` in the Makefile, I would prefer forcing `i18n` in this command so developers don't need to think about it before pushing code.

WDYT ?

Reply:
ok, both work for me